src/blink-api.js

```diff
@@ -1,6 +1,6 @@
 const crypto = require("crypto");
 // const fetch = require('node-fetch');
-const { fetch } = require( 'fetch-h2' );
+const { fetch } = require( '@adobe/helix-fetch' );
```

Comment:
There is an issue causing ERR_HTTP2_INVALID_SESSION error fixed in the adobe fork of fetch-h2 (mentioned https://github.com/grantila/fetch-h2/issues/88 and https://github.com/adobe/helix-fetch/issues/103) so it should include the benefits of fetch-h2. The adobe helix-fetch has since been re-written to version 2, but this change uses the fixed fetch-h2 fork <v2.

I didn't see this error after changing the version, however I subsequently added the error handling a couple of weeks later.